ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/CommandRunner.java

```diff
@@ -289,6 +302,9 @@ void fetchAndRunCommands() {
     lastPollTime.set(clock.instant());
     final List<QueuedCommand> commands = commandStore.getNewCommands(NEW_CMDS_TIMEOUT);
     if (commands.isEmpty()) {
+      if (!commandTopicExists.get()) {
+        commandTopicDeleted = true;
```

Comment:
we can save this for another PR, but I think it makes sense to have this method return some status then to set a ton of flags and check them in the main loop

Reply:
I'll file a follow up issue for that refactoring improvement.